ActindoWorkFlow/Operation/09_TrackWorkflowProcess.md

```diff
@@ -0,0 +1,333 @@
+
+    - **Retry**    
+    The process action failed, but will be retried after a period of time has elapsed. 
+
+    - **Error**   
+    The process action failed finally.
+<!--- zurzeit noch nicht aktiv-
+5. Check the *Profile* column to see if a profile has been created for a process action. This might have been the reason for a longer processing time. If desired, open the profile. For detailed information, see [Profile](../UserInterface/06d_Profile.md).-->
+
+5. Check the *Queue type* to see the queue type with which the process action was performed. Maybe the process action was throttled and this was the reason for a long processing time. For detailed information, see [Create a queue type](../Integration/01_ConfigureQueueTypes.md#create-a-queue-type).
+
+6. Check the *Worker number* to see the number of workers who performed the process action. 
+
+7. Check the *Transition* column to see the label and key of the transition.
+
+## Make a user decision
+
+When a process is waiting for a user interaction, such as when a business transaction requires approval, you can make a user decision to allow the workflow process to continue.       
+User decisions are implemented by the *Manual action* core action, see [Manual action](../UserInterface/08_CoreActions.md#manual-action).
+
+
+#### Prerequisites
+
+- A workflow has been created, see [Create a workflow](./01_ManageWorkflows.md#create-a-workflow).
+- A workflow process has been initiated.
+- The workflow process requires a user decision.
+
+#### Procedure
+
+*Process Orchestration > Processes > Tab Overview*
+
+![Processes overview](../../Assets/Screenshots/ActindoWorkFlow/Processes/ManualAction.png "[Processes overview]")
+
+1. Enable the ![Manual processes](../../Assets/Icons/Toggle.png "[Manual processes]") *Manual processes* toggle.    
+    Only processes requiring a manual action are displayed. They have the **In progress** status and a warning sign ![Warning](../../Assets/Icons/Warning.png) in the *Has a manual action* column.
+
+2. Click a desired process.      
+    The *Process ID* view is displayed. In the workflow diagram, the manual action is displayed in yellow.
+
+   ![Process ID](../../Assets/Screenshots/ActindoWorkFlow/Processes/ManualActionProcessID.png "[Process ID]")
+
+3. Click the checkbox of the manual action in the *Actions* tab.    
+    The editing toolbar is displayed.
+
+4. Click the [TAKE DECISION] button.   
+   The *Take decision* window is displayed.
+   
+   ![Take decision](../../Assets/Screenshots/ActindoWorkFlow/Processes/ManualActionsTakeDecision.png "[Take decision]")
+
+5. If you want the process to continue directly, click the *Select output* drop-down list, select the desired decision, and click the [SET DECISION] button.   
+    The workflow process changes to the *In progress* status and proceeds. 
```

Comment:
Vorher war schon In progress (s. Schritt 1). Wahrscheinlich ist der warning sign aber weg?
Oder meinst du "The process action changes to the "In progress" status and proceeds."?